skada/metrics.py

```diff
@@ -619,3 +619,125 @@ def _score(self, estimator, X, y, sample_domain=None):
             score = self.source_scorer(y[source_idx], y_pred_source)
 
         return self._sign * score
+
+
+class MixValScorer(_BaseDomainAwareScorer):
+    """
+    MixVal scorer for unsupervised domain adaptation.
+
+    This scorer uses mixup to create mixed samples from the target domain,
+    and evaluates the model's consistency on these mixed samples.
+
+    See [32]_ for details.
+
+    Parameters
+    ----------
+    lmbd : float, default=0.55
```

Comment:
I don't know if sklearn uses `lmbd` name. Maybe switch to alpha or mixup_param is better ?